docs-mslearn/framework/understand/ingestion.md

```diff
@@ -108,6 +108,21 @@ At this point, you have a data pipeline and are ingesting data into a central da
 
 <br>
 
+## Data Ingestion Metrics
+
+| **Category** | **Definition** | **KPI** |
```

Comment:
Can you think about the cost and carbon impact of each one of these? It may not apply everywhere. Anything that comes back to something that is metered, like data size or compute time.